main/cluster/predict.py

```diff
@@ -81,30 +77,23 @@ def predict_save(yr,model,X,y,columns=config.COLUMNS[0]):
 import csv
 import pandas as pd
 import configurations.utility as util
-# from dataManipulation.generarTablasVariables import load
 from dataManipulation.dataPreparation import getData
 from sklearn.metrics import roc_auc_score
 import numpy as np
 from sklearn import linear_model
 from main.SafeLogisticRegression import SafeLogisticRegression
 import importlib
+np.random.seed(config.SEED)
```

Comment:
I checked, and this is key for reproducible local predictions, but they are still different to those of the cluster.

Reply:
We also get reproducibility in the cluster, but not between systems.